gateway/src/main/java/org/georchestra/gateway/accounts/admin/AbstractAccountsManager.java

```diff
@@ -76,6 +80,15 @@ public GeorchestraUser getOrCreate(@NonNull GeorchestraUser mappedUser) throws D
         return find(mappedUser).orElseGet(() -> createIfMissing(mappedUser));
     }
 
+    public Optional<GeorchestraUser> findByEmailAndOAuth2OrgId(GeorchestraUser mappedUser) {
```

Comment:
missing javadoc ? Maybe it'd be too tautological though ... I am also not sure I get the intent of the method, would it be like finding an external user (e.g. authenticated via a third party identity provider or via http headers) by its email ?

Reply:
also the method has been added, but it does not look like it is called anywhere else in this PR ?

Reply:
Oh nice catch, this is a refactoring relic. I can delete this.